test/test_jit.py

```diff
@@ -561,6 +561,7 @@ def fn(x):
         torch._C._tracer_exit((y,))
         self.assertExpectedTrace(trace)
 
+    @unittest.expectedFailure
```

Comment:
What's wrong?

Reply:
Functions are traced twice because `forward()` now operates on Variables. There might be a related in-place bug too. @ezyang is fixing them in #4807

Reply:
Has that been fixed?